axelrod/tests/unit/test_plot.py

```diff
@@ -1,14 +1,12 @@
+import tempfile
 import unittest
-import axelrod
 
+import matplotlib.pyplot as plt
 from numpy import mean
-import tempfile
 
-import matplotlib
-matplotlib.use("agg")
-
-import matplotlib.pyplot as plt
+import axelrod
 
+import matplotlib
```

Comment:
should this one occur before `import axelrod`?

Reply:
Import orders don't actually matter anymore, the `matplotlibrc` ooverrides everything but I got myself confused there and will reorder to fit PEP8! (and a bonus rerun of CI to make sure it's still working :crossed_fingers:)